src/main/java/org/idpass/lite/IDPassReaderComponent.java

```diff
@@ -0,0 +1,143 @@
+package org.idpass.lite;
+
+import com.google.protobuf.ByteString;
+import io.mosip.kernel.core.util.CryptoUtil;
+import org.api.proto.Dat;
+import org.api.proto.Ident;
+import org.idpass.lite.exceptions.IDPassException;
+import org.idpass.lite.proto.PostalAddress;
+import org.springframework.stereotype.Component;
+
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReadParam;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import com.github.jaiimageio.jpeg2000.impl.J2KImageReader;
+
+/**
+ * Spring boot singleton component execution wrapper of
+ * an IDPassReader instance
+ */
+
+@Component
+public class IDPassReaderComponent
+{
+    public IDPassReader reader;
+
+    /**
+     * Instantiates IDPassReader reader with a particular configuration
+     *
+     * @throws IDPassException Standard exception
+     * @throws IOException Standard exception
+     */
+    public IDPassReaderComponent()
+            throws IDPassException, IOException
+    {
+        reader = new IDPassReader();
+        reader.setDetailsVisible(
```

Comment:
We need to move this to config.

Reply:
Yes, i agree this should be in configuration.